solvebio/contrib/dash/solvebio_auth.py

```diff
@@ -20,6 +20,7 @@ class SolveBioAuth(OAuthBase):
     AUTH_COOKIE_NAME = 'dash_solvebio_auth'
     TOKEN_COOKIE_NAME = 'solvebio_oauth_token'
 
+    DEFAULT_TOKEN_COOKIE_MAX_AGE = 60 * 60 * 24 * 7
```

Comment:
Is this obvious that it's 7 days or should I add a comment?

Reply:
I think this is definitely more obvious